lib/vanagon/project/dsl.rb

```diff
@@ -159,6 +159,17 @@ def version_from_git
         warn "Directory '#{dirname}' cannot be versioned by git. Maybe it hasn't been tagged yet?"
       end
 
+      def version_from_branch
+        branch = Git.open(File.expand_path("..", Vanagon::Driver.configdir)).current_branch
+        if branch =~ /(\d+(\.\d+)+)/
+          return $1
+        else
+          warn "Can't find a version in your branch, make sure it matches <number>.<number>, like maint/1.7.0/fixing-some-bugs"
+        end
+      rescue Git::GitExecuteError => e
+        warn "Something went wrong trying to find your git branch.\n#{e}"
```

Comment:
Eh, I've had a change of heart. I'm going to make these fail. If you're trying to get a version from git we should expect that to work.